index.html

```diff
@@ -2,12 +2,713 @@
+            id="children"
+            name="bar"
+            type="text"
+            value="0"
+          />
+          <label class="search--check__title_sm _end" for="rooms">Rooms</label>
+          <input
+            class="search--check__input_sm _end"
+            id="rooms"
+            name="bar"
+            type="text"
+            value="1"
+          />
+        </div>
+        <div class="search--submit_sm block-sm">
+          <input
+            class="search--submit__input_sm"
+            name="submit"
+            type="submit"
+            value="Search"
+          />
+        </div>
+      </form>
+      <div class="container lg-2-col sm-2-col app">
+        <svg>
+          <use href="#google-play" />
+        </svg>
+        <svg>
+          <use href="#app-store" />
+        </svg>
+      </div>
+    </div>
+    <section class="container advantages">
+      <h2 class="title">What do we offer</h2>
+      <div class="row">
+        <div class="col-lg-3 col-sm-6">
+          <div class="column advantages--row__headphones">
+            <div class="advantages--row__img">
+              <svg class="advantages--row__img-headphones">
+                <use href="#headphones" />
+              </svg>
+            </div>
+            <div class="advantages--row__title">
+              <p>Support 24/7</p>
+            </div>
+          </div>
+        </div>
+        <div class="col-lg-3 col-sm-6">
+          <div class="column advantages--row__forum">
+            <div class="advantages--row__img">
+              <svg class="advantages--row__img-forum">
+                <use href="#forum" />
+              </svg>
+            </div>
+            <div class="advantages--row__title">
+              <p>Communicate directly</p>
+            </div>
+          </div>
+        </div>
+        <div class="col-lg-3 col-sm-6">
+          <div class="column advantages--row__booking">
+            <div class="advantages--row__img">
+              <svg class="advantages--row__img-booking">
+                <use href="#booking" />
+              </svg>
+            </div>
+            <div class="advantages--row__title">
+              <p>Book online</p>
+            </div>
+          </div>
+        </div>
+        <div class="col-lg-3 col-sm-6">
+          <div class="column advantages--row__like">
+            <div class="advantages--row__img">
+              <svg class="advantages--row__img-like">
+                <use href="#like" />
+              </svg>
+            </div>
+            <div class="advantages--row__title">
+              <p>Real guest reviews</p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="background-homes">
+      <div class="container homes">
+        <h2 class="homes-title">Homes guests loves</h2>
+        <div class="homes--gallery">
+          <img src="./image/main/leopold.jpg" alt="photo of hotel leopold" />
```

Comment:
Change the paths when changing the project structure. The most of images are broken.